### PR TITLE
file to get the details for the post into groups

### PR DESCRIPTION
**Pull request by iharshit009**

### CODE REVIEW

**Review by thealphadollar — CHANGES_REQUESTED** [review]
This is an amazing PR. Thanks a lot for it :sparkle: 

I would request you to keep all this in the folder `src` which has the __init__.py file and delete entries such as `fb_login.py`, `open_group.py` and others which we won't be using :)

Also segregate different input in different methods.

For e.g. accept username and password in one function, and details for post in another.

Again, this is a great PR and almost finishes the skeleton of the project.

**Comment by iharshit009** [comment]
Sir, please have a check at the PR

File: src/accept_data.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        self.info()
+
+    def fb_details(self):
+        self.username = input("enter your username: ")
+        self.password = getpass.getpass("enter your password: ")
+
+    def group_details(self):
+        self.group = input("please enter group pages URL \
+        seperated through ',' : ")
+        self.group = self.group.split(',')
```
Comment (thealphadollar) [inline src/accept_data.py]: self.groups would be a better name

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Fabulous work :) I'm highly thankful to you for this.

Please implement the last set of changes and then we shall move on with more.

Also, please rename the file `src/accept_data.py` to `app.py` or `main.py`

**Review by thealphadollar — APPROVED** [review]
Thanks a lot.

LGTM :sparkles:

**Pull request merged**